### PR TITLE
news: hexo-renderer-jade 0.5.0 & hexo-renderer-pug 1.0.0

### PR DESCRIPTION
**Pull request by curbengh**
- [x] Others (Update, fix, translation, etc...)

### CODE REVIEW

File: source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+and rename all `*.jade` files to `*.pug`.
+
+## hexo-renderer-jade 0.5.0
+
+### Breaking change
+- Final release, this plugin has been deprecated and replaced by [hexo-renderer-pug](https://github.com/hexojs/hexo-renderer-pug) [#35]
+- chore: drop node 6 [#29]
+
+### Dependency
```
Comment (SukkaW) [inline source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md]:
```suggestion
### Dependencies
```

File: source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Official plugins hexo-renderer-jade 0.5.0 & hexo-renderer-pug 1.0.0 released
+---
+
+Since the upstream [jade] package has been replaced by [pug] package and no longer maintained, we've decided to deprecate [hexo-renderer-jade](https://github.com/hexojs/hexo-renderer-jade) and replace it with [hexo-renderer-pug].
```
Comment (NoahDragon) [inline source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md]: Should we also add link for hexo-rednerer-pug?

Comment (SukkaW) [reply source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md]: The link of `hexo-renderer-pug` is already added.

@curbengh is using a markdown syntax called `reference links` and the URL is at L44.

The preview of this PR can be found [here](https://deploy-preview-1185--hexo-site.netlify.com/news/2019/10/19/hexo-renderer-jade-0-5_hexo-renderer-pug-1/).

Comment (NoahDragon) [reply source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md]: Thank you @SukkaW . The preview is so cool. 😁 

I think we should keep the format consistent. The hexo-renderer-jade is using the direct link, but the hexo-renderer-pug is the reference.

File: source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+-  "hexo-renderer-jade": "^0.4.1",
++  "hexo-rederer-pug": "^1.0.0",
+```
+
+and rename all `*.jade` files to `*.pug`.
+
+## hexo-renderer-jade 0.5.0
+
+### Breaking change
+- Final release, this plugin has been deprecated and replaced by [hexo-renderer-pug](https://github.com/hexojs/hexo-renderer-pug) [#35]
```
Comment (SukkaW) [inline source/_posts/2019-10-19-hexo-renderer-jade-0-5_hexo-renderer-pug-1.md]:
```suggestion
- Final release, this plugin has been deprecated and replaced by [hexo-renderer-pug] [#35]
```

Is it ok to use `reference links` syntax?

**Comment by curbengh** [comment]
fixed links

**Pull request merged**